libmozdata/HGFileInfo.py

```diff
@@ -58,6 +58,9 @@ def get(self, path, utc_ts_from=None, utc_ts_to=None, authors=[]):
 
         for entry in entries:
             assert self.date_type in entry
+            if isinstance(entry[self.date_type], six.string_types):
```

Comment:
Can you clone mozilla-central again and verify that this is actually an issue?
If it is, then we should at least print a warning here to avoid failing silently if something changes on hg.mozilla.org.

(What value are you seeing when this is not a list?)